frontend/views/pages/PayGroup.vue

```diff
@@ -142,11 +188,49 @@ export default {
             amount: 50
           }
         ],
-        currency: currencies['USD']
+        currency: currencies.USD.symbol
       }
     }
   },
   computed: {
+    ...mapGetters([
+      'groupMembers',
+      'groupSettings'
+    ]),
+    mincomeAmount () {
+      return this.groupSettings.mincomeAmount
+    },
+    currency () {
+      return currencies[this.groupSettings.mincomeCurrency]
+    },
+    ourUsername () {
```

Comment:
usually we use `currentUsername`. I believe we should pay more attention to our naming, we have a lot of _similar-but-to-equal_ names across `currentUsername, ourUsername, loggedUsername, currentName`, etc...

Reply:
created `ourUsername` getter and replaced all instances of `this.$store.state.loggedIn.username` with it